doc/package.md

```diff
@@ -0,0 +1,176 @@
+                        contains the server content required to execute JAXRS applications -->
+                        <layer>jaxrs-server</layer>
+                    </layers>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>package</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            ...
+        </plugins>
+        ...
+    </build>
+...
+</project>
+```
+
+## Fine tune server configuration and package extra content
+
+The example below shows how to configure the plugin goal to execute JBoss CLI commands and copy extra content inside the installed server: 
+
+```xml
+<project>
+    ...
+    <build>
+        ...
+        <plugins>
+            ...
+            <plugin>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>${project.artifactId}</artifactId>
+                <version>${project.version}</version>
+                <configuration>
+                    <feature-packs>
+                        <feature-pack>
+                            <location>org.jboss.eap:wildfly-ee-galleon-pack</location>
+                        </feature-pack>
+                        <feature-pack>
+                            <location>org.jboss.eap.cloud:eap-cloud-galleon-pack</location>
+                        </feature-pack>
+                    </feature-packs>
+                    <layers>
+                        <layer>jaxrs-server</layer>
+                    </layers>
+                    <packaging-scripts>
+                        <!-- Commands and scripts are executed in the context of an embedded-server. 
+                        You don't need to start and stop the embedded-server. -->
+                        <execution>
+                            <!-- List of commands -->
+                            <commands>
+                                <command>/system-property=org.jboss.example.runtime:write-attribute(name=dev)</command>
+                                <command>/system-property=org.jboss.example.runtime2:write-attribute(name=dev2)</command>
+                            </commands>
+                            <!-- Properties files containing System Properties to resolve expressions -->
+                            <properties-files>
+                                <file>cli.properties</file>
+                            </properties-files>
+                            <!-- Enable expression resolution prior to send the commands to the server, false by default -->
+                            <resolve-expressions>true</resolve-expressions>
+                            <!-- List of CLI script files -->
+                            <scripts>
+                                <script>config.cli</script>
```

Comment:
it'd be better to use the standard location for scripts: this should be

```
<!-- CLI scripts are located in src/main/scripts directory -->
<script>${project.build.scriptSourceDirectory}/config.cli</script>
```

Reply:
Good suggestion. Fixed.